### ISSUE TITLE
Django 3.x has no six included

### ISSUE THREAD

**Issue by Dmitri-Sintsov** [niwinz/django-jinja#236]
```
  File "/home/user/work/djk-sample3.6/lib/python3.6/site-packages/django/apps/config.py", line 116, in create
    mod = import_module(mod_path)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/user/work/djk-sample3.6/lib/python3.6/site-packages/django_jinja/apps.py", line 4, in <module>
    from django_jinja import base
  File "/home/user/work/djk-sample3.6/lib/python3.6/site-packages/django_jinja/base.py", line 11, in <module>
    from django.utils import six
ImportError: cannot import name 'six'

```

**Comment by Dmitri-Sintsov** [comment]
Created https://github.com/niwinz/django-jinja/pull/237

**Comment by ShaheedHaque** [comment]
I've just run into this too...

**Comment by palfrey** [comment]
For reference, this was fixed 819a5faddea5341d990ac337b05e7827ac5b4431 and the 2.5.0 release
